src/antelope/stores/contract.ts

```diff
@@ -13,27 +13,146 @@
 
 import { defineStore } from 'pinia';
 import {
-    createInitFunction,
     createTraceFunction,
+    isTracingAll,
+    useFeedbackStore,
 } from 'src/antelope/stores/feedback';
+import EvmContract, { Erc20Transfer } from 'src/antelope/stores/utils/contracts/EvmContract';
+import EvmContractFactory from 'src/antelope/stores/utils/contracts/EvmContractFactory';
+import { useChainStore } from 'src/antelope';
+import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
+import { getTopicHash, TRANSFER_SIGNATURES } from 'src/antelope/stores/utils';
+import { AntelopeError, EvmContractFactoryData, EvmTransaction } from 'src/antelope/types';
+import { ethers } from 'ethers';
 
 
-export interface ContractState {
-    abi: string;
+export interface ContractStoreState {
+    factory: EvmContractFactory;
+    cachedContracts: Record<string, EvmContract>
+    processing: string[];
```

Comment:
There's nothing wrong with this code, it is just not using the same '__' prefix as the other stores do to easily differentiate the actual data variables from the rest of the auxiliary functions and properties. This facilitates debugging and reading.

would you mind renaming those variables across the file?

```suggestion
    __factory: EvmContractFactory;
    __cachedContracts: Record<string, EvmContract>
    __processing: string[];
```